react/package.json

```diff
@@ -179,6 +181,10 @@
         "webpack-conditional-loader": "^1.0.12",
         "webpack-dev-server": "^4.11.0"
     },
+    "peerDependencies": {
+        "react": "^17 || ^18",
```

Comment:
Support of react 18 will only be valid once there is no dependency on @material-ui any more :/

Reply:
I'm not good enough to answer :/
But I tried that out:
- I could update your branch to react 18 without problem (I was surprised)
- `npm ls react` fails because of missing dependencies to react 17
- running the storybook fails to load all the subsurface and well log stories :(

I double checked, and @material-ui does not allow react 18 in its peer dependencies.